legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-execution/legend-engine-xt-relationalStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/plugin/RelationalExecutionNodeExecutor.java

```diff
@@ -2155,6 +2159,20 @@ private static DoubleStrategyHashMap<Object, Object, SQLExecutionResult> switche
         return parentMap;
     }
 
+    private static DoubleStrategyHashMap<Object, Object, SQLExecutionResult> switchedParentHashMapPerChildResult(RelationalGraphObjectsBatch relationalGraphObjectsBatch, int parentIndex, ResultSet childResultSet, Supplier<List<String>> parentPrimaryKeyColumnsSupplier, boolean isDatabaseIdentifiersCaseSensitive)
```

Comment:
I dont see the need for a new function like this. Ideally all usages of the function need to be changed. And the new function should be something like

private List<String> upperCaseColumnsIfDbConnectionIsCaseSensistive(List<String>, DatabaseConnection)

Reply:
done